test/dynamics/common.py

```diff
@@ -81,3 +102,20 @@ def jit_grad_wrap(self, func_to_test: Callable) -> Callable:
         wf = wrap(lambda f: jit(grad(f)), decorator=True)
         f = lambda *args: np.sum(func_to_test(*args)).real
         return wf(f)
+
+
+class TestQutipBase(unittest.TestCase):
+    """Base class with setUpClass and tearDownClass for setting jax as the
```

Comment:
This is good, but update the doc string to indicate that it's for qutip. The `tearDownClass` isn't necessary for the qutip case, so remove this from the doc string as well.